src/main/java/homework_nr_11/HomeWorkNr11.java

```diff
@@ -0,0 +1,60 @@
+package homework_nr_11;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class HomeWorkNr11 {
+    public static void main ( String[] args ) {
+
+        List<Person> personList = getPeople();
+
+
+        System.out.println( "The persons which name starts with letter A and is less than 5 characters " );
+        List<Person> nameStartsWithLetterA = personList.stream()
+                .filter( e -> e.name.startsWith( "A" ) )
+                .limit( 5 )
```

Comment:
this is going to limit the amount of objects the stream will find. It has nothing to do with the string itself being a size smaller than 5